Dockerfile

```diff
@@ -46,8 +47,8 @@ VOLUME [ "/var/lib/docker" ]
 ENTRYPOINT [ "/usr/local/share/docker-init.sh" ]
 CMD [ "sleep", "infinity" ]
 
+RUN apt-get update -y
```

Comment:
the commands from lines 50 - 53 can be combined in a single RUN layer

Reply:
we do that intentionally so that the layers pull in parallel and startup is faster. There's a comment on line 16